recipes/ee_extra/LICENSE

```diff
@@ -0,0 +1,194 @@
+Apache License
```

Comment:
You don't need to include this file since it's already present in the PyPI tarball. Please remove.

Reply:
The license file has been removed :)